apps/atomy-q/API/app/Http/Controllers/Api/V1/DecisionTrailController.php

```
@@ -122,4 +141,174 @@ public function export(Request $request): JsonResponse
+                'artifact_kind' => 'award_workflow_action',
+                'description' => match ($entry->event_type) {
+                    'award_created' => 'Award created',
+                    'award_debriefed' => 'Award debrief drafted',
+                    'award_signed_off' => 'Award signed off',
+                    default => $entry->event_type,
+                },
+            ]);
+        }
+
+        return $base;
+    }
```

PHPMD (2.15.0)

[warning] The method trailMetadata() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)